src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -420,24 +428,26 @@ class ContentSidebar extends PureComponent<Props, State> {
                 <aside id={this.id} className={styleClassName}>
                     <div className="be-app-element">
                         {hasSidebar ? (
-                            <APIContext.Provider value={(this.api: any)}>
-                                <Sidebar
-                                    file={((file: any): BoxItem)}
-                                    view={view}
-                                    detailsSidebarProps={detailsSidebarProps}
-                                    activitySidebarProps={activitySidebarProps}
-                                    metadataSidebarProps={metadataSidebarProps}
-                                    getPreview={getPreview}
-                                    getViewer={getViewer}
-                                    hasSkills={hasSkills}
-                                    hasDetails={hasDetails}
-                                    hasMetadata={hasMetadata}
-                                    hasActivityFeed={hasActivityFeed}
-                                    onToggle={this.onToggle}
-                                    onVersionHistoryClick={onVersionHistoryClick}
-                                    currentUser={currentUser}
-                                />
-                            </APIContext.Provider>
+                            <LoggerContext.Provider value={(this.logger: any)}>
```

Comment:
I think this will need to live in the consuming app, such as EUA. Some developers may not want or need our logs.